Source/OxyPlot.WindowsForms/GraphicsRenderContext.cs

```diff
@@ -311,8 +311,7 @@ public override void DrawRectangle(OxyRect rect, OxyColor fill, OxyColor stroke,
 
                 this.g.TranslateTransform(dx, dy);
 
-                var layoutRectangle = new RectangleF(0, 0, size.Width + 0.1f, size.Height + 0.1f);
-                this.g.DrawString(text, font, fill.ToBrush(), layoutRectangle, this.stringFormat);
+                this.g.DrawString(text, font, fill.ToBrush(), 0.0f, 0.0f, this.stringFormat);
```

Comment:
no more support for clipped text? We want to keep that feature - see the example with a long title that is clipped